pysphero/device_api/power.py

```diff
@@ -75,6 +84,16 @@ def wake(self):
 
         self.request(PowerCommand.wake)
 
+    def get_battery_state_LMQ(self) -> BatteryLMQStates:
+        """
+        Get battery state without known voltage constants
+
+        :return BatteryVoltageStates:
+        """
+
+        response = self.request(PowerCommand.get_battery_state_LMQ, timeout=100)
+        return BatteryLMQStates(response.data[0]).name
```

Comment:
Maybe return `BatteryLMQStates(response.data[0])`?

Reply:
I was looking to avoid the class name with this.
If I put BatteryLMQStates(response.data[0]) return will contain "BatteryLMQStates.charged"
whereas if BatteryLMQStates(response.data[0]).name return will contain "charged"

Reply:
BatteryLMQStates(response.data[0]) is BatteryLMQStates, but BatteryLMQStates(response.data[0]).name is string. 
Getting BatteryLMQStates seems more helpful for me..

Reply:
Valid point, updated